### ISSUE TITLE
报错了

### ISSUE THREAD

**Issue by lhy2749** [CementMaker/cnn_lstm_for_text_classify#2]
TypeError:batch_normalization() missing 5 required positional arguments: 'mean', 'variance', 'offset', 'scale', and 'variance_epsilon'
###
self.outputs, self.final_state = tf.nn.dynamic_rnn(lstm_cell, embedded, dtype=tf.float32, time_major=False)
            self.out = tf.unstack(tf.nn.batch_normalization(self.outputs), axis=1)[-1]
是因为现在的tensorflow方法修改了么···

**Comment by vicdu** [comment]
@lhy2749 我也有相同的错误，请问你解决了吗

**Comment by xuefeng199507** [comment]
> @lhy2749 我也有相同的错误，请问你解决了吗

把均值方差自己设一下就好了

**Comment by CementMaker** [comment]
两天之内代码会重新更新和修改，tensorflow很多api更改了，最近考虑更改一下
